flepimop/gempyor_pkg/src/gempyor/compartments.py

```diff
@@ -898,14 +1017,20 @@ def list_recursive_convert_to_string(thing):
 @cli.group()
 @pass_context
 def compartments(ctx: Context):
-    """Commands for working with FlepiMoP compartments"""
+    """
+    Add commands for working with FlepiMoP compartments.
+    A container for subcommands `plot` and `export`.
+    Commands in this container will require context about the compartmental model.
+    """
```

Comment:
I'm not sure how much sense this will make to users when doing `flepimop compartments --help`? Not like the previous version was helpful either, but I doubt users will know what containers and context are when using the CLI.